client/src/components/Survey.tsx

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { Link, Stack } from '@fluentui/react';
+import { PostCallConfig } from '../models/ConfigModel';
+import { rejoinLinkStyle, surveyStyle } from '../styles/Survey.styles';
+
+export interface SurveyProps {
+  onRejoinCall: () => void;
+  postCall?: PostCallConfig;
+}
+const SURVEY = 'SurveyComponent';
+
+export const Survey: React.FunctionComponent<SurveyProps> = (props: SurveyProps) => {
+  return (
+    <Stack styles={surveyStyle}>
+      <iframe
+        title={SURVEY}
+        style={{ height: '100%', width: '100%', border: 0 }}
```

Comment:
I think it would better to move this to separate variable for consistency.

Reply:
Thanks, have made this change